web/src/hooks/transaction/useCheckInRun.ts

```diff
@@ -25,15 +23,17 @@ const useCheckInRun = (fields: CheckInFields, onSuccess?: () => void) => {
       functionName: 'checkIn',
       args: [
         fields.challengeId,
-        fields.timestamp,
-        fields.v,
-        `0x${fields.r.padStart(64, '0')}`,
-        `0x${fields.s.padStart(64, '0')}`,
+        numberToHex(fields.activityId).padEnd(32, '0'), //checkIn data, we use activityId
```

Comment:
Should we also adjust the CheckIn log in contract? Log this instead of timestamp?

Reply:
yes!

Reply:
will open another PR and fix it in the next release